ansible_base/lib/abstract_models/common.py

```diff
@@ -136,7 +136,7 @@ def from_db(self, db, field_names, values):
 
         return instance
 
-    def get_summary_fields(self):
+    def get_summary_fields(self, view=None):
```

Comment:
Can you give more details here as to why view is needed? It's never referenced in this function or passed anywhere else unless I am missing something.

Reply:
It's required for `user_capabitilies`. In AWX, the `get_summary_fields` are placed in serializer, which has `self.context.get('view', None)`, but DAB's code calls `obj.get_summary_fields` which targets model, so the view has to be propagated there. 

Anyway this feature was postponed, so I'm converting it to draft